<?xml version="1.0"?>
<launch>
  <include file="$(find racecar_gazebo)/launch/racecar.launch">
	<arg name="world_name" value="track_levine"/>
  </include>
    <include file="$(find racecar)/launch/includes/common/joy_teleop.launch.xml" >
      <arg name="racecar_version" value="racecar-v2" />
  </include>

  # output="screen" allows us to see print statements
  <node pkg="lidart_wall_following" name="pid_error_node" type="pid_error.py" output="screen"/>
  <node pkg="lidart_wall_following" name="control_node" type="control.py" output="screen"/>
  <node pkg="lidart_wall_following" name="lidart_analysis_node" type="lidart_analysis.py" output="screen"/>
  <node pkg="lidart_wall_following" name="sim_connector_node" type="sim_connector.py" output="screen"/>

</launch>
